homework/homework_part_2/JS/app.js

```diff
@@ -0,0 +1,105 @@
+console.log("hello");
+//-------------PART 1: TERMS------------------//
+// What is the difference between interpolation and concatenation? Give an example of each.
+// -- concatenation is when you use + to combine strings together. interpolation is when you use ${string} to combine strings.
```

Comment:
specifically its when one of the strings you are combinging (whether with a `+` or using the ES7 `${}` syntax) was stored in a variable